src/main/java/IssuePage.java

```diff
@@ -0,0 +1,105 @@
+import helper.Level;
+import org.junit.jupiter.api.Assertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+import static helper.ColorPrinter.printColorMessage;
+
+public class IssuePage extends BasePage {
+    private final WebElement assigneesField = driver.findElement(By.xpath("//summary[@data-menu-trigger='assignees-select-menu']"));
+    private final WebElement labelsField = driver.findElement(By.xpath("//summary[@data-menu-trigger='labels-select-menu']"));
+    private final WebElement projectsField = driver.findElement(By.xpath("//summary[@data-menu-trigger='projects-select-menu']"));
+    private final WebElement milestonesField = driver.findElement(By.xpath("//summary[@data-menu-trigger='milestone-select-menu']"));
+    private final static String TITLE = "IssuePage";
+
+    IssuePage(WebDriver driver) {
+        super(driver, TITLE);
+    }
+
+    public void addIssueWithAllFieldsAndComment(String issueTitle, String IssueDescription, String AssigneeXPath, String LabelXPath, String ProjectXPath, String MilestoneName, String commentText) {
+        printColorMessage("Adding Issue with all Fields filled And Comment added has been started.", log, Level.INFO);
+        WebElement titleField = driver.findElement(By.xpath("//input[@placeholder='Title']"));
+        webDriverWait = new WebDriverWait(driver, Duration.ofMillis(1000));
+        titleField.sendKeys(issueTitle);
+        WebElement descriptionField = driver.findElement(By.id("issue_body"));
+        descriptionField.sendKeys(IssueDescription);
+        chooseAssignee(AssigneeXPath);
+        chooseLabels(LabelXPath);
+        chooseProjects(ProjectXPath);
+        chooseMilestones(MilestoneName);
+        submitNewIssueCreation();
+        addComment(commentText);
+        printColorMessage("Adding Issue with all Fields filled And Comment added has been finished.", log, Level.INFO);
+
+    }
+
+    public void chooseAssignee(String AssigneeNameXPath) {
+        webDriverWait = new WebDriverWait(driver, Duration.ofMillis(1000));
```

Comment:
is this waiter really needed here? if test passes without it you can remove this waiter